README.md

```diff
@@ -66,12 +66,12 @@ You will need to install the Protobuf Compiler. This can be done by executing:
 sudo apt install -y protobuf-compiler
 ```
 
-### <a name="install-fontconfig-dev-library">Install fontconfig-dev library</a>
+### <a name="install-sdl2-library">Install SDL2 library</a>
 
-You will need to install the fontconfig-dev library. This can be done by executing:
+You will need to install the libsdl2-dev library. This can be done by executing:
 
 ```shell
-sudo apt install -y libfontconfig-dev
+sudo apt install libsdl2-dev
```

Comment:
consider leaving the `-y` argument for convenience

Reply:
Added.